packages/ui/src/app/pages/WorkingGroups/WorkingGroup/OpeningsTab.tsx

```diff
@@ -22,6 +23,7 @@ export const OpeningsTab = ({ workingGroup }: Props) => {
   })
   const { member: leader } = useMember(workingGroup.leaderId)
   const { workers } = useWorkers(workingGroup.id ?? '')
+  const currentOpenings = useMemo(() => openings.filter(isOpeningOpen), [isLoading, openings])
```

Comment:
Let's move filtering to the hook by either

1.  Hook parameter `useOpenings( { type: 'PAST' | 'CURRENT' } )`
2.  Different hook, like `useCurrentOpenings()` and `usePastOpenings()`